src/h3lib/include/h3api.h.in

```diff
@@ -165,6 +165,18 @@ typedef struct {
     GeoPolygon *polygons;
 } GeoMultiPolygon;
 
+/**
+ * Values representing polyfill containment modes, to be used in
+ * the `flags` bit field for `polygonToCellsExperimental`.
+ */
+typedef enum {
+    CONTAINMENT_CENTER = 0,       ///< Cell center is contained in the shape
+    CONTAINMENT_FULL = 1,         ///< Cell is fully contained in the shape
+    CONTAINMENT_OVERLAPPING = 2,  ///< Cell overlaps the shape at any point
+    CONTAINMENT_OVERLAPPING_BBOX = 3,  ///< Cell bounding box overlaps shape
+    CONTAINMENT_INVALID = 4  ///< This mode is invalid and should not be used
+} ContainmentMode;
```

Comment:
Definitely not blocking, just a thought: do we want to label this like `ContainmentModeExperimental`?

Reply:
I don't feel strongly we should do so. This does not seem to be part of the experimental API (except maybe overlapping_bbox) and should be easier to change because it is not part of the ABI.